vaadin-radio-button-flow-integration-tests/src/main/java/com/vaadin/flow/component/radiobutton/tests/RefreshDataProviderPage.java

```diff
@@ -33,12 +33,12 @@ public RefreshDataProviderPage() {
         group.setId("group");
 
         List<String> items = new LinkedList<>(Arrays.asList("foo", "bar"));
-        group.setDataProvider(new ListDataProvider<>(items));
+        group.setItems(new ListDataProvider<>(items));
```

Comment:
Why not just `group.setItems(items);` ?

Reply:
I could simply change it but I thought it is better to keep changes as minimal as possible on already existing codes. please let me know whether it is OK to change old codes.

Reply:
Yes, we can change this code, taking into account that passing items is better that create a new data provider. And also the name of page and test also should be changed. Now something like 'RefreshItemsPage' would be better, because data provider removed